apis/deployer/healthchecks/types_healthchecks.go

```diff
@@ -0,0 +1,63 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package healthchecks
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/selection"
+
+	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+)
+
+// HealthChecksConfiguration contains the configuration for health checks.
+type HealthChecksConfiguration struct {
+	// DisableDefault allows to disable the default health checks.
+	// +optional
+	DisableDefault bool `json:"disableDefault,omitempty"`
+	// Timeout is the time to wait before giving up on a resource to be healthy.
+	// Defaults to 180s.
+	// +optional
+	Timeout *lsv1alpha1.Duration `json:"timeout,omitempty"`
+	// CustomHealthChecks is a set of custom health check configurations
+	// +optional
+	CustomHealthChecks []CustomHealthCheckConfiguration `json:"customHealthChecks,omitempty"`
```

Comment:
```suggestion
	CustomHealthChecks []CustomHealthCheckConfiguration `json:"custom,omitempty"`
```
I think `healthchecks` might be duplicated as we are already in the healtcheck configuration struct